src/scss/dashboard.scss

```diff
@@ -0,0 +1,122 @@
+@import "settings";
+.dashboard {
+    width: 100%;
+    max-width: 100%;
+}
+
+.dashboard--scrollable {
+    height: 70vh;
+    overflow: scroll;
+}
+
+.dashboard__body {
+    height: calc(100vh - 4em);
+}
+
+.dashboard__logo {
+    height: 3em;
+    position: absolute;
+    left: 1.5em;
+    top: 1em;
+}
+
+.dashboard__header {
+    height: 4em;
+}
+
+.dashboard__greet {
+    color: $white !important;
+    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+    float: right;
+    margin: 0;
+    margin-right: 2em;
+    line-height: 4em
+}
+
+.dashboard__title {
+    font-size: 1.2em;
+    line-height: 1em;
+    padding-bottom: 0.5em;
+    padding-top: 0.5em;
+    color: #FFFFFF;
+    font-weight: bold;
+    font-family: Helvetica Neue, Helvetica, Arial, sans-serif;
+    position: absolute;
+    right: 7%;
+    top: 0.5em;
+}
+
+.dashboard__sidebar {
+    padding: 0;
+    li {
+        border-bottom: 1px solid $white;
+        margin: 0;
+        height: 4.5em;
+        line-height: 4.5em;
+    }
+    .dashboard__sidebar__item {
```

Comment:
Use only a single ```__elem``` per selector see [here](http://getbem.com/faq/#what-would-be-a-class-name-for-an-element-inside-another-element-block__elem1__elem2-)